softwerkskammer/lib/groupsAndMembers/groupsAndMembersService.js

```diff
@@ -100,6 +99,18 @@ module.exports = {
     });
   },
 
+  getOrganizersOfGroup: function (groupId, callback) {
+    this.getGroupAndMembersForList(groupId, (error, groupIncludingMembers) => {
+      if (error) {return callback(error);}
+      if (!groupIncludingMembers) {
+        return callback(null, []);
+      }
+      const groupMembers = groupIncludingMembers.members || [];
```

Comment:
inline plz

Reply:
`const organizers = (groupIncludingMembers.members || []).filter(member => groupIncludingMembers.isOrganizer(member.id()));`?

Reply:
Ne ne ne, jetzt muss ich Euch beiden mal widersprechen. Einfach mal nen Schritt zurücktreten.

Also, wozu brauchst Du den `|| []` Fall? Wenn es zwar groupIncludingMembers gibt, aber members nicht.

OK, wenn wir den Fall mal im Auge behalten: Was kommt dann in der nächsten Zeile raus? `organizers = []`. Denn eine leere Liste bleibt unter Anwendung eines Filters immer leer.

Was passiert jetzt mit dieser leeren Liste? Sie wird im Callback zurückgegeben.

Haben wir das schonmal irgendwo in dieser Form gemacht? Hmm... Hmm... Ja! In Zeile 106.

Also was ist hier die einfachste Lösung? Die Prüfung in Zeile 105 zufügen. Dann unten groupIncludingMembers.members inlinen. Und die Organizers auch noch. :-)

(ich nenne das "die dünnste Stelle für eine Änderung suchen").

Reply:
Ich hab mir das gerade nochmal angeschaut: der Fall, dass `members` undefiniert ist, kann es wohl nicht geben: bei einem entsprechenden Test schlägt bereits das bauen der Member-Liste in `addMembersToGroup` fehl und wenn man sich die Tests des `memberstore` anschaut, dann scheint bei einer leeren Mailingliste immer ein leeres Array zurückzukommen. Daher würde ich das `|| []` einfach rausschmeißen, oder?

Reply:
Jo, dann passt das, denke ich. Dann kannst Du auch über ein Inlining nachdenken ;-)

Reply:
done